### PR TITLE
Bug 1870362 - Add underline to each link in the add-on details view

### PR DESCRIPTION
**Pull request by willdurand**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1870362

### CODE REVIEW

**Comment by willdurand** [comment]
After:

image

**Comment by willdurand** [comment]
@anna-yeddi is that what you had in mind to fix https://bugzilla.mozilla.org/show_bug.cgi?id=1870362? Thanks

File: fenix/app/src/test/java/org/mozilla/fenix/addons/AddonDetailsBindingDelegateTest.kt [hunk 1]
```diff
@@ -85,7 +85,7 @@ class AddonDetailsBindingDelegateTest {
                 ratingUrl = "https://example.org/",
             ),
         )
-        assertEquals("100", binding.reviewCount.text)
+        assertEquals("100", binding.reviewCount.text.toString())
```
Comment (Amejia481) [inline fenix/app/src/test/java/org/mozilla/fenix/addons/AddonDetailsBindingDelegateTest.kt:88]: Should we addition cases to verify we have the underline added ?

**Review by Amejia481 — APPROVED** [review]
Added a non blocking question, in general looks good!

**Comment by anna-yeddi** [comment]
> @anna-yeddi is that what you had in mind to fix https://bugzilla.mozilla.org/show_bug.cgi?id=1870362? Thanks

That's perfect @willdurand ! Thank you for fixing that!

File: fenix/app/src/test/java/org/mozilla/fenix/addons/AddonDetailsBindingDelegateTest.kt (hunk 1 above)
Comment (willdurand) [reply fenix/app/src/test/java/org/mozilla/fenix/addons/AddonDetailsBindingDelegateTest.kt:88]: as discussed on slack, I am not sure how we can cover that with an assertion :(

**Pull request merged**